site/content/docs/main/website-guidelines.md

```diff
@@ -13,21 +13,21 @@ At the root of the project, run:
 make serve-docs
 ```
 
-This runs all the Ruby dependencies in a container.
+This runs all the Hugo dependencies in a container.
 
 Alternatively, for quickly loading the website, under the `velero/site/` directory run:
 
 ```bash
-bundle exec jekyll serve --livereload --future
+hugo serve
 ```
 
-For more information on how to run the website locally, please see our [jekyll documentation](https://github.com/vmware-tanzu/velero/blob/main/site/README-JEKYLL.md).
+For more information on how to run the website locally, please see our [Hugo documentation](https://gohugo.io/getting-started/).
 
 ## Adding a blog post
 
-The `author_name` value must also be included in the tags field so the page that lists the author's posts will work properly (Ex: https://velero.io/tags/carlisia%20campos/). 
+The `author_name` value must also be included in the tags field so the page that lists the author's posts will work properly (Ex: https://velero.io/tags/carlisia%20campos/).
```

Comment:
"for example"

Reply:
@ashish-amarnath thanks for pointing this out. I actually realized that the instructions were out of date and updated them for the new hugo way of adding a blog